js/pages/samples.js

```diff
@@ -313,26 +314,25 @@ const sanford = {
-    whatHappens: 'Check in at the front desk when you enter the Imagenetics Building. The registration team will direct you to the lab where the Connect team will collect your samples. At the end of your visit, the Connect team will check you out of your appointment.<br><br>We will draw a blood sample, collect a urine sample, and collect a saliva sample by asking you to swish with mouthwash.<br><br>We will also ask you to complete a short survey on the MyConnect app using your mobile phone. You will need your MyConnect app login information to complete the survey. If you do not have a mobile phone, we may be able to provide you with a tablet to complete your survey.<br><br>We strongly encourage you to complete your survey at your appointment. If you choose to complete it after you leave your appointment, it is important to do so as soon as possible.',
+    whatHappens: 'Check in at the registration desk. The registration team will direct you where to go next in order to get your samples collected.  At the end of your visit, the Connect team will check you out of your appointment.<br><br>We will draw a blood sample, collect a urine sample, and collect a saliva sample by asking you to swish with mouthwash.<br><br>We will also ask you to complete a short survey on the MyConnect app using your mobile phone. You will need your MyConnect app login information to complete the survey. If you do not have a mobile phone, we may be able to provide you with a tablet to complete your survey.<br><br>We strongly encourage you to complete your survey at your appointment. If you choose to complete it after you leave your appointment, it is important to do so as soon as possible.',
     support: 'Call 1-877-505-0253 (8:00 a.m.-10:00 p.m. CT on weekdays and 9:00 a.m.-6:00 p.m. CT on weekends)',
     locations: [
         [
-            'Sioux Falls: Sanford Imagenetics',
+            'Sanford Imagenetics',
```

Comment:
```suggestion
            'Sioux Falls: Sanford Imagenetics',
```
It looks like the doc says: `Sioux Falls: Sanford Imagenetics` Please correct me if I'm wrong. https://nih.app.box.com/file/805344518276?s=e5xonzmm27rvd77crt1mnrbalit51a7v

Reply:
I reviewed the new doc - please update this per the suggestion.